### PR TITLE
lua-copas: Update to 2.0.2

### PR DESCRIPTION
**Pull request by neheb**
Switched to codeload + other adjustments.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @srdgame 
Compile tested: ar71xx

### CODE REVIEW

File: lang/lua-copas/patches/makefile.patch [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
--PREFIX = /usr/local
-+PREFIX ?= $(DESTDIR)/usr
+-PREFIX  ?= /usr/local
++PREFIX  ?= $(DESTDIR)/usr
  
  # System's lua directory (where Lua libraries are installed)
--LUA_DIR= $(PREFIX)/share/lua/5.1
-+LUA_DIR= $(PREFIX)/lib/lua
+-LUA_DIR ?= $(PREFIX)/share/lua/5.1
++LUA_DIR ?= $(PREFIX)/lua/5.1
```
Comment (karlp) [inline lang/lua-copas/patches/makefile.patch]: Is this patch even needed?  that default won't work anyway, so it must be getting properly set by the openwrt toolchain already.

Comment (neheb) [reply lang/lua-copas/patches/makefile.patch]: Nope. good call.

File: lang/lua-copas/Makefile [hunk 2]
```diff
@@ -8,18 +8,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lua-copas
-PKG_VERSION:=2.0.0
+PKG_VERSION:=2_0_2
```
Comment (karlp) [inline lang/lua-copas/Makefile]: I'm still a little concerned by this version style, are you sure it plays well withopkg-compare-versoins?

Comment (neheb) [reply lang/lua-copas/Makefile]: Beats me. I have no plans to backported this to 17.01 or 18.06.

Comment (neheb) [reply lang/lua-copas/Makefile]: Upstream tags it like this in git. I could open an issue.

**Comment by hnyman** [comment]
> this would be similar to specifying PKG_SOURCE_VERSION:=xx_yy_zz

That would be preferrable, as the main PKG_VERSION would stay normal.

**Comment by neheb** [comment]
But then the issue would be two variables instead of 1. I'll ask upstream for a solution.

**Comment by neheb** [comment]
Upstream has provided a solution. Using the normal version format directly now.

**Pull request merged**